### ISSUE TITLE
Bench should run on multiple Crux configurations

### ISSUE THREAD

**Issue by jarohen** [xtdb/xtdb#574]
We'd like to run each bench against multiple Crux configurations:

* tx-log: both standalone and embedded-kafka
* kv-store: memdb, rocksdb

Minus lmdb for now until https://github.com/juxt/crux/issues/280 is resolved

* for each configuration, we should add dimensions for the tx-log/kv-store, so that we can see which configuration it ran against

**Comment by jarohen** [comment]
Failed overnight 2020-02-25

**Comment by t-taylor** [comment]
We've decided to remove `memdb` from the benchmark configurations as currently we run out of memory on our aws benchmarking system. We also believe that `memdb` shouldn't be used in production and is primarily for dev work as an easy non-persistent crux setup. This card is however blocked by https://github.com/juxt/crux/issues/664. So for the moment will be moved back into `Selected`

**Comment by jarohen** [comment]
given we've separated LMDB out into a separate card, will close this one
